docs/application/vstools/tools/project-wizard.md

```diff
@@ -1,53 +1,53 @@
 ﻿# Project Wizard
 
-The Project Wizard in Visual Studio Tools for Tizen is used to create a Tizen .NET application project.
+You can use Project Wizard to create Tizen .NET application projects. In Project Wizard, you can select a template and a profile to create the project. Aditionally, you can set the basic project properties, such as the project name, package name, location, and so on.
 
-In the Project Wizard, you can select a template and profile to create the project. In addition, you can set basic project properties, such as the project name, package name, and location.
+The Project Wizard provides the following project templates using which you can start a Tizen application project:
 
-The Project Wizard provides 4 project templates with which you can start a .NET application project:
+Template | Description | Supported Versions
+-------- | ----------- | -------------------
+**Blank App (ElmSharp-Beta)** | Use this template to create Tizen applications using the ElmSharp APIs. | Tizen 4.0, Tizen 5.0
```

Comment:
The reason for adding "the" is because in the "using the ElmSharp APIs" sentence, APIs acts as a qualifier here and we add "the" to support the qualifier. 
Similarly for "the Open Toolkit library", here library acts as a qualifier.

Reply:
If it is your decided policy about how to control "the", than I'll refer.